phrosty/pipeline.py

```diff
@@ -151,7 +153,7 @@ def get_psf( self, ra, dec, dump_file=False ):
         #   PSF.  We need to fix that... somehow....
 
         if self.keep_intermediate:
-            dump_file = True
+            save_file = True
```

Comment:
I'd be happier if this `keep_intermediate` check were done before the call to `get_psf`.  I don't like two different ways of setting this `save_file` keyword.